src/api/custom-resolvers/submit-application.ts

```diff
@@ -21,14 +22,17 @@ const submitApplication = async (root: any, variables: SubmitApplicationVariable
     })) as Application;
 
     if (application) {
-      const canSubmit = application.status === APPLICATION.STATUS.DRAFT;
+      const hasDraftStatus = application.status === APPLICATION.STATUS.DRAFT;
```

Comment:
will this be needed anywhere else? could it go in a helper

Reply:
It won't be used anywhere other than in here - we could extract, but then we'd have duplicated unit tests, I don't think it's worth doing